packages/component-library/components/Notification/components/GenericNotification.tsx

```diff
@@ -130,6 +131,12 @@ class GenericNotification extends React.Component<Props, State> {
     )
   }
 
+  textContainerClassName(): string {
+    return classnames(styles.textContainer, styles[this.props.style], {
```

Comment:
Actually, I'm just looking at the failed snapshot tests and are we sure we want the `styles[this.props.style]` part? 

Side note: did snapshots actually catch a real issue for once?

Reply:
Nope we don't need it! Sorry, getting use to the snapshots still 😅